### ISSUE TITLE
Block names from saconfig

### ISSUE THREAD

**Issue by rovertdude** [TotalFreedom/TotalFreedomMod#1490]
This might be useful for possibly blocking a part of the family policy.
This would make a config part of names that won't be supered.
This can also work for some rogue admins who can cause permanent damage that won't get re-supered.

**Comment by RobinGall2910** [comment]
Not sure if its possible. @JeromSar

**Comment by MrImmaculate** [comment]
I see what you mean. You want to have a list of usernames that are exempt from being given admin privileges. Smart but, what robin said factors in.

**Comment by JeromSar** [comment]
Obviously possible, but what you wish to accomplish needs to be more properly defined. Do you want admins to only be supered if the username matches? Do you want a global list of usernames which will never be added when using /saconfig add?

**Comment by DUFCLiam** [comment]
YES, They cannot do that command if name is in config, or a new .yml file called "superbanned.yml" or something like that if not we can implement this in "config.yml"

**Comment by rovertdude** [comment]
Like @DUFCLiam said; it's a list of names that won't be supered even if their IP matches in the saconfig.
